pages/seed/getting-started/overview.mdx

```diff
@@ -2,30 +2,65 @@ import { Tabs, Tab } from "nextra/components";
 
-Snaplet is an **all-in-one developer tool** that gives developers **realistic, production-like data** that they can code against.
+**Seed** is a set of tools **understanding your database schema** to help you **generate production-accurate data**.
 
-There are two ways to obtain production-like data with Snaplet: either by [capturing](/snapshot/core-concepts/capture) an existing source database into a **snapshot**, or by [generating](/seed/core-concepts) fake data using AI.
+It consists of the following parts:
 
-<iframe width="100%" height="500" src="https://www.youtube.com/embed/-MfItFmmeqU?si=Yzg41rK4Spoy7FHa" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+- **Seed ORM**: Auto-generated and type-safe data client for Node.js & TypeScript 🛡️
+- **Seed CLI**: A CLI to generate and keep the data client in sync with your database 🛠️
+- **Seed AI**: Our custom model identifying the shape of your data 🤖
 
-### Seed
-If you don't have access to an existing database that has production or production-like data, Snaplet's [seed](/core-concepts/seed) feature allows you to create production-like fake data using AI.
+We are compatible with [PostgreSQL]() and [SQLite](). MySQL support is coming soon.
 
-## Quick Start
+## Seed ORM
 
-Follow the [`seed` quick start](/getting-started/quick-start/seed) to get seed data using AI. This is the recommended path if you don't have a database you can connect to. If you do have access to a database and want production-like data, check out our [snapshot quick start](/getting-started/quick-start/snapshot) guide.
+Our data client is auto-generated from your database schema.
+We are using the full power of TypeScript to provide you with a type-safe and auto-completed experience when working with your data.
 
-## Key Features
+You can focus on the data that matter to **you**, we will take care of generating all the relationships. ✨
 
-- **Seed data:** Create production-like fake data using AI.
+Here is a sneak peek of what you can do with Seed ORM:
 
-## Learn Snaplet
+Are you working on a blog app and need 5 posts?
+```ts
+await seed.posts((x) => x(5))
+```
 
-Dive into the key concepts of Snaplet:
+Or maybe you need a user with a specific email?
+```ts
+await seed.users([
+  { email: 'snappy@snaplet.dev' },
+])
+```
 
-📚 [Seed](/seed/core-concepts) fake data using AI.
+What if you want to create an organization with a varying number of members?
+```ts
+await seed.organizations([
+  {
+    name: 'Snaplet',
+    members: (x) => x({ min: 1, max: 10 }),
+  },
+])
+```
+
+And this is just the beginning! Explore the [Seed ORM guide]() to see all the possibilities.
```

Comment:
Where would this link to?

Reply:
I think we will take what we have in "core-concept".